### ISSUE TITLE
exp gene expression dynamics

### ISSUE THREAD

**Issue by szb37** [KarrLab/wc_model_gen#3]
In model_gen’s transcription submodel generator, we should an option to generate a submodel with ‘exponential_gene_expression’ dynamics. Everything should remain the same as right now, with the exception of the rate law which should be: (ln2/[cell’s doubling_time])*[concentration of the RNA being transcribed]. In parallel with this, there should be an option for exponential degradation as well, where we have RNA’s the half_life instead on the doubling_time in the expression. Down the line, we should have exponential translation as well, but that is low priority at the moment.
